lib/blackjack_methods.rb

```diff
@@ -1,5 +1,3 @@
-require 'pry'
-
```

Comment:
You shouldn't leave pry hanging around in your deployed code.

```ruby
require 'pry'; binding.pry;
```

Is how I do it in my code. You can just drop that line instead of binding.pry and remove the whole line when you are done with it. Also it makes it fail to run if you don't have pry installed.